docs/components/authentication/sign-in.mdx

```diff
@@ -99,12 +99,15 @@ All props below are optional.
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `appearance` | `object` | Controls the overall look and feel of the component. See [Appearance](/docs/components/customization/overview) for more information. |
-| `routing` | `string` | The routing strategy for your pages.<br />Supported values are: <ul><li>hash (default): Hash-based routing.</li><li>path: Path-based routing.</li><li>virtual: Virtual-based routing.</li></ul> |
-| `path` | `string` | The path where the component is mounted on when path-based routing is used e.g. /sign-up. Note: If you are using Environment Variables for Next.js or Remix to specify your routes, this will be set to `path`.|
```

Comment:
Oops it looks like this note was on the wrong prop. We should keep the note but move it to the `routing` row.